bigquery/bq_schema.json

```diff
@@ -52,7 +52,7 @@
   },
   {
     "mode": "REQUIRED",
```

Comment:
Also need to update `MH` to be nullable, right?

Reply:
Yep I made MH, BCH, BSP all nullable, those are the main values that display numbers and hold weird numbers in the case of an error

Reply:
Also no worries, I won't merge until given the ok to merge. Thank you for letting me know

Reply:
> Yep I made MH, BCH, BSP all nullable, those are the main values that display numbers and hold weird numbers in the case of an error

My bad, didn't realize these were already nullable. Thanks.